creativecloud/features/interactive-components/slider-tray/slider-tray.js

```diff
@@ -173,7 +187,39 @@ function sliderEvent(media, layer) {
   });
 }
 
-function uploadImage(media, layer) {
+function cssToPhotoshop(imgObj, adjustment, value) {
+  const unitValue = convertToUnit(adjustment, value, CSSRanges);
+  imgObj[adjustment] = convertFromUnit(adjustment, unitValue, PsRanges);
+}
+
+function convertToUnit(adjustment, value, ranges) {
+  if (value < ranges[adjustment].min || value > ranges[adjustment].max) {
+    throw new Error(`value out of range ${adjustment}:${value}`);
+  }
+
+  if (value < ranges[adjustment].zero) {
+    const spread = ranges[adjustment].zero - ranges[adjustment].min;
+    return (value - ranges[adjustment].min) / spread - 1;
+  }
+  const spread = ranges[adjustment].max - ranges[adjustment].zero;
+  return (value - ranges[adjustment].zero) / spread;
+}
+
+function convertFromUnit(adjustment, value, ranges) {
+  if (value < -1 || value > 1) {
+    throw new Error(`value out of range ${adjustment}:${value}`);
```

Comment:
Can we return a default here in place of throwing error?

Reply:
done